### ISSUE TITLE
Zooming out animation

### ISSUE THREAD

**Issue by ghost** [ashare80/TSClusterMapView#22]
A little detail to the behaviour of leaf annotations turning to clustered annotations while zooming out of the map:
It would be great if the leaf annotations would stay as they are but move to the mutual position and afterwards turn into the clustered annotation. Right now the annotations are already changed to clustered annotations (including number and UIImage) and then they are moved to the mutual position and all except one get deleted.

**Comment by AdamShare** [comment]
@architector1 Yes right now the clustering operation already assigns all the annotations their new cluster objects before the animation begins and calls the delegate to update their views.  The reason for this is because some view that are off screen would not be the correct view prior to animating on screen.

I can look into being more selective and better handling this situation.

**Comment by trungnguyen1791** [comment]
Hi @ashare80 .
Thanks for great lib.
I have 1 question related to this issue. Can I somehow force clusters to split manually? I mean, split a few level without zooming .
Thanks.
